ssas/systems.go

```diff
@@ -443,7 +443,8 @@ func GetSystemByID(id string) (System, error) {
 	)
 	defer Close(db)
 
-	if err = db.First(&system, id).Error; err != nil {
+	// must use the explicit where clause here because the id argument is a string
```

Comment:
Would it be beneficial to convert the string to `uint` first, to validate before hitting the database?